### DISCUSSION TITLE
offline storage for blobs in native mobile apps built using Quasar

### DISCUSSION THREAD

**Discussion by markbruk** [quasarframework/quasar#14888] [CLI - Capacitor / Cordova modes]
I am building an app that creates mp3 recordings that need to be stored locally in the device.  I built my app using IndexedDB for the blobs and localstorage for the keys, and this worked great.  Then I compiled the app for Android and found that IndexedDB only works within browser builds but not native apps.  I have not yet tested iOS or Windows or Mac, my other 3 target platforms, but Android is my largest target market and so I am starting here and I am stuck here.

My app uses HTML5 record to create an end user recording of a word that must then be stored in the app as the app must work offline, where there is no connectivity after the initial download and install of my app.

I would appreciate finger pointing to any technology that might lead me to success.  Thanks in advance

**Comment by metalsadman** [comment]
use filesystem https://capacitorjs.com/docs/apis/filesystem
